en/app.ftl

```diff
@@ -1143,3 +1143,17 @@ forwarded-email-footer-offer-countdown-banner = Upgrade to { $premium_link } bef
 ## Notifications component
 
 toast-button-close-label = Close notification
+
+## API error messages
+
+# Variables:
+#   $free_tier_limit (string) - Maxmimum email masks created for free account
+# api-error-free-tier-limit = You must be a premium subscriber to make more than { $free_tier_limit } email masks.
```

Comment:
is this a comment or a string?

Reply:
It was a comment. This PR was still in draft mode. Disregarding comment for now.